internal/peer/common/ordererclient.go

```diff
@@ -22,11 +23,12 @@ type OrdererClient struct {
 
 // NewOrdererClientFromEnv creates an instance of an OrdererClient from the
 // global Viper instance
-func NewOrdererClientFromEnv() (*OrdererClient, error) {
+func NewOrdererClientFromEnv(tlsHandshakeTimeShift time.Duration) (*OrdererClient, error) {
```

Comment:
Not saying that the viper approach is a great one, but it's a little odd that all of the parameters for setting up the ordering connection are wired in via `configFromEnv`, but not the timeshift.  And, it's also pretty ugly that everyone calling for this function has to explicitly disable the timeshift by passing in a zero value.  I don't know that it gets any less ugly following the existing pattern, but thought I'd ask?

Reply:
Yeah, I definitely don't love this change. I did it this way because I hadn't realized the viper config that `configFromEnv` was using actually derives from flags that are then `viper.Set` in `peer/common/ordererenv.go`'s `SetOrdererEnv()`. I'll update things to be consistent with the others now that I realize that.

Reply:
Fixed